packages/mdc-menu-surface/util.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016 Google Inc. All Rights Reserved.
+ * Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
While we're here, I think there are a bunch of these held over from old JS animation logic in "Simple" Menu that we don't need anymore (they're only referenced in tests):

* `clamp`
* `bezierProgress` (and `getBezierCoordinate_` and `solvePositionFromXValue_`)